DOCUMENTATION.md

```diff
@@ -85,11 +85,11 @@ If you feel something is missing, not clear or could be improved, please don't h
             * [.device](#resin.models.environment-variables.device) : <code>object</code>
                 * [.getAll(uuidOrId)](#resin.models.environment-variables.device.getAll) ⇒ <code>Promise</code>
                 * [.getAllByApplication(nameOrId)](#resin.models.environment-variables.device.getAllByApplication) ⇒ <code>Promise</code>
-                * [.create(uuidOrId, name, value)](#resin.models.environment-variables.device.create) ⇒ <code>Promise</code>
+                * [.create(uuidOrId, envVarName, value)](#resin.models.environment-variables.device.create) ⇒ <code>Promise</code>
                 * [.update(id, value)](#resin.models.environment-variables.device.update) ⇒ <code>Promise</code>
                 * [.remove(id)](#resin.models.environment-variables.device.remove) ⇒ <code>Promise</code>
             * [.getAllByApplication(applicationNameOrId)](#resin.models.environment-variables.getAllByApplication) ⇒ <code>Promise</code>
-            * [.create(applicationNameOrId, name, value)](#resin.models.environment-variables.create) ⇒ <code>Promise</code>
+            * [.create(applicationNameOrId, envVarName, value)](#resin.models.environment-variables.create) ⇒ <code>Promise</code>
```

Comment:
These changes mean the docs weren't rebuilt last time. Can you please make the build command be `gulp build && npm run docs`?